_posts/2022-12-23-trino-summit-2022-upsolver-recap.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Using Trino to analyze a product-led growth (PLG) user activation funnel"
+author: "Mei Long, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/upsolver.jpg
+---
+
+As the holiday season approaches, we have reached the end of our
+[Trino Summit 2022 recap posts]({% post_url 2022-11-21-trino-summit-2022-recap %}).
+With the last talk of the summit, Mei Long from Upsolver gave an insightful
+overview of how they use data to inform product decisions.
+
+<!--more-->
+
+{% youtube MCB_1furnAo %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@Upsolver.pdf">
+  Check out the slides!
+</a>
+
+## Recap
+
+When talking about product-led growth, it helps to start by defining what it
+even means. The core idea is simple: see how users engage with your product,
```

Comment:
semicolon not colon

Reply:
Colon is correct here. Semicolon works, too, but would be a little more strange.